example/purchase-tester/ios/App/App.xcodeproj/project.pbxproj

```diff
@@ -353,7 +353,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = App/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.6;
```

Comment:
Probably not a big deal, but perhaps this should match the iOS version in the Podfile? (iOS 15)